src/services/ui/src/utils/crumbs.ts

```diff
@@ -1,12 +1,26 @@
-import { BreadCrumbConfig, Route } from "@/components";
-import { mapActionLabel, mapSubmissionCrumb } from "@/utils";
+import { BreadCrumbConfig, Route, urlEmbedQuery } from "@/components";
+import { authorityById, mapActionLabel, mapSubmissionCrumb } from "@/utils";
 import { Action } from "shared-types";
 
-export const dashboardCrumb: BreadCrumbConfig = {
-  displayText: "Dashboard",
-  order: 1,
-  default: true,
-  to: "/dashboard",
+export const dashboardCrumb = (id?: string): BreadCrumbConfig => {
+  if (id) {
+    const authority = authorityById(id);
+    const newPath = urlEmbedQuery("/dashboard", {
+      tab: authority === "" ? "spas" : authority,
+    });
+    return {
+      displayText: "Dashboard",
+      order: 1,
+      default: true,
+      to: newPath,
+    };
+  }
+  return {
+    displayText: "Dashboard",
+    order: 1,
+    default: true,
+    to: "/dashboard",
+  };
```

Comment:
this can be cleaned up quite a bit to reduce the double-return:

```javascript
return {
    ...
    to: id ? 
          urlEmbedQuery("/dashboard", { tab: !authorityById(id).length ? "spas" : authorityById(id) }) 
          : "/dashboard"
}
```

Reply:
this could be further simplified by simplifying the interface of authorityById (see below)